### PR TITLE
Removed dataspace and allow custom configured Humio URL

### LINKED ISSUE #7: Remove dataspace parameter and parametize URL
As per [our conversation](https://chaostoolkit.slack.com/archives/C9BS95CQM/p1578750085003900).

The `dataspace` (which now is called a repository) is not used anymore.
https://github.com/chaostoolkit-incubator/chaostoolkit-humio/blob/e7757200d9b9844135fddd8cefd74af2465a5e63/chaoshumio/__init__.py#L34-L37

The `url` var
https://github.com/chaostoolkit-incubator/chaostoolkit-humio/blob/e7757200d9b9844135fddd8cefd74af2465a5e63/chaoshumio/__init__.py#L44-L46
should be expose to the user for on-prem installation.
The new repository/dataspace-less url is [`/api/v1/ingest/humio-ingest`](https://docs.humio.com/api/ingest/#structured-data).

My suggestion is to replace parametrize the `https://cloud.humio.com` part of the current `url` var and completely ignore `dataspace`.

### PR DESCRIPTION
**Pull request by russmiles**
Added Humio Custom URL as configuration to control and notification if specified

Removed dataspace secret as no longer used in Humio url

Closes #7

Signed-off-by: Russ Miles <russ@russmiles.com>

### CODE REVIEW

**Comment by russmiles** [comment]
@mwl Can you give this code a glance and a spin before I merge it into master?

**Comment by russmiles** [comment]
@gtfisher can you give it a spin without the HUMIO_DATASPACE as that should not be required any more?

**Comment by gtfisher** [comment]
Checked without the HUMIO_DATASPACE, still looks good

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@
 
 [Unreleased]: https://github.com/chaostoolkit-incubator/chaostoolkit-humio/compare/0.3.0...HEAD
 
+### Added
+
+-   Overriding of the Humio URL domain to support on-premise deployments of Humio.
```
Comment (mwl) [inline CHANGELOG.md]: 🤣 https://collectivecontent.co.uk/2018/04/19/should-i-say-on-premise-or-on-premises-it/

**Pull request merged**